assets/src/edit-story/elements/media/edit.js

```diff
@@ -162,6 +164,9 @@ function MediaEdit({ element, box }) {
     cropMediaProps.srcSet = srcSet;
   }
 
+  const borderProps =
+    canMaskHaveBorder(element) && borderRadius ? { borderRadius } : null;
```

Comment:
Can it have a border-radius specified, if it has a mask not supporting it?

Reply:
If everything works as expected then not. I guess it can just check for `borderRadius` then.